libraries/AP_Generator/AP_Generator_IE_FuelCell.cpp

```diff
@@ -144,16 +142,16 @@ const AP_Generator_IE_FuelCell::Lookup_State AP_Generator_IE_FuelCell::lookup_st
 };
 
 // Check for any change in error state or status and report to gcs
-void AP_Generator_IE_FuelCell::check_status()
+void AP_Generator_IE_FuelCell::check_status(const uint32_t now)
 {
     // Check driver health
-    if (!healthy() && !_health_warn_sent) {
+    if (!healthy() && (!_health_warn_last_ms || (now - 20000 >= _health_warn_last_ms))) {
```

Comment:
I think you need to change this to 
```suggestion
    if (!healthy() && (!_health_warn_last_ms || (now - _health_warn_last_ms >= 20000))) {
```

Reply:
It's gives the same outcome.  But happy to change if there is an AP convention.

Reply:
currently it wraps in the first 20 seconds of boot, the other way will not wrap.

Reply:
the new way wraps correctly, hence it is also an Ardupilot Convention

Reply:
Thanks both.  I had not thought of that.  Changed as requested.